### DISCUSSION TITLE
Help connecting to Google Drive with linux cli

### DISCUSSION THREAD

**Discussion by tmadms** [iterate-ch/cyberduck#14317] [Announcements]
I'm trying to connect with my Google Drive on the CLI but I'm not sure how to proceed. After entering `duck --list googledrive:/<user>`
I get a message to 'Open web browser to authenticate and obtain an authorization code'. I press the link and follow the instructions on the web but in the shell it is not accepting an input. Am I missing something?

**Comment by dkocher** [comment]
We track this problem in #13587.
